### DISCUSSION TITLE
Output ANSI escape sequence in log

### DISCUSSION THREAD

**Discussion by suntong** [go-kit/kit#1157] [Q&A]
Hi, 

First of all, I know there is a `NewColorLogger`, but I just want to avoid it and do ANSI escape sequence myself. 

Now, please take a look
https://gist.github.com/suntong/a558d24b7209bc62ee932bbfbe31c6b1

its output is not what I was expecting:

![image](https://user-images.githubusercontent.com/422244/103247470-03c0e680-4935-11eb-9e02-9f83c1c10513.png)

Any way to get my own ANSI escape sequence output works? thx

**Comment by peterbourgon** [comment]
I believe this is a function of the logfmt format, and likely by design — you don't want unescaped escape sequences in your logs by default, as they can be [an exploit vector](https://ma.ttias.be/terminal-escape-sequences-the-new-xss-for-linux-sysadmins/).

**Comment by suntong** [comment]
Oh, thx. 

if I don't use `NewLogfmtLogger` but a different one, say the term or file ones, would any of them works then?
